mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -31,8 +33,14 @@ def _check_rank_input(rank, data, indices):
             if "copy" in inspect.getfullargspec(data.get_data).kwonlyargs:
                 kwargs["copy"] = False
             data_arr = data.get_data(**kwargs)
-        else:
-            data_arr = data
+        elif isinstance(data, (EpochsSpectrum, EpochsSpectrumArray)):
+            # Spectrum objs will drop bad channels, so specify picking all channels
+            data_arr = data.get_data(picks=np.arange(data.info["nchan"]))
```

Comment:
Realised that `Spectrum.get_data(picks=None)` returns only good channels, but the same for `Epochs` returns all channels, so need to explicitly pick all channels from `Spectrum` objects to keep behaviour consistent.